task01/src/main.cpp

```diff
@@ -0,0 +1,7 @@
+
+#include "Vector.h"
+
+int main()
+{

```

Comment:
Next exercises please make sure to add a `main.cpp` that tests your code. Currently you have no way of knowing if your implementation works